src/SoftwareLicense.php

```diff
@@ -826,17 +772,17 @@ public static function cronSoftware($task = null)
                 $options['licenses']    = $items;
 
                 if (NotificationEvent::raiseEvent('alert', new self(), $options)) {
-                    $entityname = Dropdown::getDropdownName("glpi_entities", $entity);
+                    $entityname = Dropdown::getDropdownName(Entity::getTable(), $entity);
                     if ($task) {
                         //TRANS: %1$s is the entity, %2$s is the message
                         $task->log(sprintf(__('%1$s: %2$s') . "\n", $entityname, $message));
                         $task->addVolume(1);
                     } else {
-                        Session::addMessageAfterRedirect(sprintf(
+                        Session::addMessageAfterRedirect(htmlspecialchars(sprintf(
```

Comment:
`$message` contains a `<br>` tag for each licence. These `<br>` tags should not be encoded.